testing/go/set_test.go

```diff
@@ -31,108 +31,6 @@ func TestSetStatements(t *testing.T) {
 // it returns an error as those parameters can only be updated
 // through configuration file and/or SIGHUP signal and/or having appropriate roles.
 var setStmts = []ScriptTest{
-	{
```

Comment:
Were these wrong? They were removed but it doesn't seem like they were relocated anywhere

Reply:
Oops, I was debugging something and forgot to put it back 😅